.github/CONTRIBUTING.md

```diff
@@ -1,338 +1,22 @@
-<div align="center">
-    <img src="https://github.com/PreMiD.png?size=2048" width="128px" style="max-width:100%;">
-    <h3 style="font-size: 2rem; margin-bottom: 0">Presence Guidelines</h3>
-    <h4 style="margin-top: 0">Revision 3</h4>
-    <br />
-</div>
+# Contrubutions
 
-# Guidelines
+As an open-sourced project, we welcome contributions with open arms. 
 
-When publishing Presences to this GitHub, we require you to follow a set of guidelines. To some, these strict rules may seem harsh. However, the implementation of these rulesets will keep us and the users from running into any issues.
+> You should ensure you read the entirety of this document before contributing to the repository. We require the following rules for good practice. 
```

Comment:
why is this a quote? Also the phrasing of this is very weird, specifically in the "you should ensure" and "We require the following rules" parts